frontend/TestRun/Components/ResultTable.svelte

```diff
@@ -19,6 +35,11 @@
         "table-secondary": "gray",
     };
 
+    const dispatch_run_click = (e) => {
```

Comment:
Wrong function naming convention

Reply:
it's natural for a python programmer :grin: